examples/schnorr.c

```diff
@@ -0,0 +1,119 @@
+/*************************************************************************
+ * Copyright (c) 2020-2021 Elichai Turkel                                *
+ * Distributed under the CC0 software license, see the accompanying file *
+ * EXAMPLES_COPYING or https://creativecommons.org/publicdomain/zero/1.0 *
+ *************************************************************************/
+
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+
+#include "random.h"
+#include "secp256k1.h"
+#include "secp256k1_extrakeys.h"
+#include "secp256k1_schnorrsig.h"
+
+
+int main(void) {
+    unsigned char msg_hash[32] = {0}; /* This should be a hash of the message. */
+    unsigned char seckey[32];
+    unsigned char randomize[32];
+    unsigned char auxiliary_rand[32];
+    unsigned char serialized_pubkey[32];
+    unsigned char signature[64];
+    int is_signature_valid;
+    secp256k1_xonly_pubkey pubkey;
+    secp256k1_keypair keypair;
+    /* The docs in secp256k1_extrakeys.h above the `secp256k1_keypair_create`
```

Comment:
That's remarkably specific, and may go outdated if the header files are restructured a bit.

What about "The specification in secp256k1_extrakeys.h states that secp256k1_schnorrsig_verify needs ..."?